### PR TITLE
Updated product pricing and added new pictures

### PR DESCRIPTION
**Pull request by PHIHOD**
Updated documentation for marketplace product presentation, further outlining pricing models.

### CODE REVIEW

**Comment by SigurdMW** [comment]
Ok, I ran it through the validation and it looks good. However, there are some spelling issues. Could you run all the text through Grammarly or similar and fix spelling and grammar? 
Also, I think it's best that @SynneLarsen approves this PR once it's ready.

**Comment by PHIHOD** [comment]
@SigurdMW Fixed grammar errors.

File: sections/marketplace/choosingyourpricingmodel.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+author: Veracity
+description: Choosing the right pricing model for your product can be challenging. Here we introduce some of the most common ones.
```
Comment (GIUCA111) [inline sections/marketplace/choosingyourpricingmodel.md:3]: after "common ones" ---> insert "that Veracity offers you."

File: sections/marketplace/choosingyourpricingmodel.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Subscription model
+
+Subscription is a well-known model and it’s easily understandable for customers. A subscription model can be set up to be just one single subscription in which case you might not capture the entire revenue potential of your service, so using a tiered subscription is advised. When making a tiered subscription you should not make too many tiers as this will make it difficult for the customers to select the one most suitable for them.​
+
+Subscription models make the cost predictable for the customer and the revenue predictable for you as the provider of the service.​
+
+Some additional mechanisms can be considered when offering subscriptions; can you offer your customers a limited trial period? Why not let your customers try out what you can offer?​
+
+Setting the price for the top tier offering for large enterprises could be difficult so you might keep this tier negotiable and making it possible for potential customers to request a price quote here.
```
Comment (GIUCA111) [inline sections/marketplace/choosingyourpricingmodel.md:18]: Change all sentence "setting the price" to ---> "Price setting for the top tier (usually offered to large enterprises) could be difficult. You might keep this tier negotiable and leave the option for the potential customer to request a price quote."

File: sections/marketplace/choosingyourpricingmodel.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+From the provider side, this model makes it easy to ensure that the consumption price covers your cost. However sporadic use by customers may make revenue unpredictable.​
+
+The price customers will pay is unpredictable and likewise, the revenue of your service will be unpredictable​
+
+Some might combine a subscription model with a consumption model where you have some included consumption in the subscription but charge overage use according to consumption.​
+
+## Freemium model​
+
+In a freemium model, the entry-level for the customer is low as the service is free. Usually, the free version has limitations in functionality or some other limitation that should trigger the customers need to upgrade to a paid version.​
```
Comment (GIUCA111) [inline sections/marketplace/choosingyourpricingmodel.md:37]: limitation ----> limitations

File: sections/marketplace/sellingyourproductoverview.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+author: Veracity
+description: "Introducing the different solutions for how to present your content and pricing most optimally."
+---
+
+# Selling your product
+
+When presenting your service in Veracity marketplace, you can choose to provide it as a free service, a paid service or a lead-generating service (service that is trying to gain the interest of potential customers to increase future sales).​
+On the marketplace, we also offer hybrid solutions, that for example include both a paid version and a free version, or a specific price plan together with a customized price solution.​
+
+In the following paragraph, we will cover each of the different solutions in detail on how to present your content and pricing most optimally.
```
Comment (GIUCA111) [inline sections/marketplace/sellingyourproductoverview.md:11]: Delete "in detail"

File: sections/marketplace/paidservices.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+
+# Paid services
+
+When planning to sell a service through Veracity Marketplace, you should have reached a high grade of maturity of your product and be ready for scalability. Get in touch with us if you want to know more about how "ready you are for selling digitally".
+
+The following section gives an overview of the different pricing models you can use for your product.
+
+## Standard price plan - horizontal visualization​
+
+Standard pricing plans, with a horizontal view of the different pricing, are used when the provider needs to sell different  “Product variants”. A product variant describes different software solutions or packages of online software/APIs, data sets.​
```
Comment (GIUCA111) [inline sections/marketplace/paidservices.md:14]: horizontal view of the different pricing ----> prices

**Pull request merged**